service/controller/v3/controllercontext/context.go

```diff
@@ -0,0 +1,48 @@
+package controllercontext
+
+import (
+	"context"
+
+	"github.com/giantswarm/microerror"
+)
+
+type contextKey string
+
+const controllerKey contextKey = "controller"
+
+type Context struct {
+	APILBBackendPoolID  string
+	EtcdLBBackendPoolID string
+	MasterSubnetID      string
+	WorkerSubnetID      string
+}
```

Comment:
We need a controller context now to dispatch information from the `deployment` resource to the `instance` resource.